src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -98,6 +100,16 @@ - (nullable instancetype)initWithAddress:(NSString *)address {
       _address = address;
       _secure = YES;
       kHostCache[address] = self;
+
+      if (!connectivityMonitor) {
+        connectivityMonitor =
+            [GRPCConnectivityMonitor monitorWithHost:hostURL.host];
```

Comment:
Ok, but then you need to (1) document that, so people don't go "huh??", and (2) think about concurrent writes and reads to that static variable.